plugins/vpp/aclplugin/data_resync_test.go

```diff
@@ -15,19 +15,20 @@
 package aclplugin_test
 
 import (
+	"testing"
+
 	"git.fd.io/govpp.git/adapter/mock"
-	govppapi "git.fd.io/govpp.git/api"
+	"git.fd.io/govpp.git/api"
```

Comment:
Why not keep the alias same for all packages?

Reply:
I see I missed some. If you find more, please leave a comment